build.sh

```diff
@@ -139,6 +139,7 @@ build_cygwin()
       --sysconfdir="${SYSCONFDIR}" \
       --localstatedir="${LOCALSTATEDIR}" \
       --datarootdir="${DATAROOTDIR}" \
+      --datarootdir="${DATADIR}"\
```

Comment:
```suggestion
      --datarootdir="${DATADIR}" \
```
keep the spaces consistent...